central/auth/datastore/datastore_impl.go

```diff
@@ -144,25 +143,23 @@ func (d *datastoreImpl) InitializeTokenExchangers() error {
 		return err
 	}
 
-	if features.PolicyAsCode.Enabled() {
-		kubeSAIssuer, err := d.issuerFetcher.GetServiceAccountIssuer()
-		if err != nil {
-			return fmt.Errorf("Failed to get service account issuer: %w", err)
-		}
-
-		// Unconditionally add K8s service account exchanger.
-		// This is required for config-controller auth.
-		configs = append(configs, &storage.AuthMachineToMachineConfig{
-			Type:                    storage.AuthMachineToMachineConfig_KUBE_SERVICE_ACCOUNT,
-			TokenExpirationDuration: "1m",
-			Mappings: []*storage.AuthMachineToMachineConfig_Mapping{{
-				Key:             "sub",
-				ValueExpression: fmt.Sprintf("system:serviceaccount:%s:config-controller", env.Namespace.Setting()),
-				Role:            "Configuration Controller",
-			}},
-			Issuer: kubeSAIssuer,
-		})
-	}
+	kubeSAIssuer, err := d.issuerFetcher.GetServiceAccountIssuer()
+	if err != nil {
+		return fmt.Errorf("Failed to get service account issuer: %w", err)
+	}
+
+	// Unconditionally add K8s service account exchanger.
+	// This is required for config-controller auth.
+	configs = append(configs, &storage.AuthMachineToMachineConfig{
+		Type:                    storage.AuthMachineToMachineConfig_KUBE_SERVICE_ACCOUNT,
+		TokenExpirationDuration: "1m",
+		Mappings: []*storage.AuthMachineToMachineConfig_Mapping{{
+			Key:             "sub",
+			ValueExpression: fmt.Sprintf("system:serviceaccount:%s:config-controller", env.Namespace.Setting()),
```

Comment:
Can we use a var up top for better code readability and have this:
ValueExpression: configControllerServiceAccountName or something shorter?